lib/Sema/CSDiagnostics.h

```diff
@@ -1255,7 +1255,10 @@ class MissingMemberFailure : public InvalidMemberRefFailure {
   /// overload to be present, but a class marked as `@dynamicCallable`
   /// defines only `dynamicallyCall(withArguments:)` variant.
   bool diagnoseForDynamicCallable() const;
-  
+
+  void diagnoseUnsafeCxxMethod(SourceLoc loc, ASTNode anchor, Type baseType,
```

Comment:
Could you please add a small comment there describing what this method is trying to do?